test/functional/e2e/login.e2e-spec.ts

```diff
@@ -0,0 +1,56 @@
+import puppeteer from 'puppeteer';
+import LoginPage from './login-page-object';
+
+const email = 'admin@alkem.io';
+const password = process.env.AUTH_TEST_HARNESS_PASSWORD;
+const loginPage = new LoginPage();
+
+describe('Authentication smoke tests', () => {
+  let browser: puppeteer.Browser;
+  let page: puppeteer.Page;
+  beforeAll(async () => {
+    browser = await puppeteer.launch({
+      headless: true,
+      //slowMo: 10,
```

Comment:
is this needed?

Reply:
Not really. The default state us headless, hence - removed.